CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 5.11.0 (unreleased version)
```

Comment:
the release should be 5.1.0 as per semantic versioning.

Reply:
Thanks. Done